Tests/Unit/EmogrifierTest.php

```diff
@@ -1519,4 +1519,77 @@ public function emogrifyAutomaticallyClosesUnclosedTag()
             $this->subject->emogrify()
         );
     }
+
+    /**
+     * @test
+     */
+    public function emogrifyHandleNoImportant()
```

Comment:
Is this test any different from the existing tests? If not, it should be removed.